.github/workflows/release-drafter.yml

```diff
@@ -12,6 +12,7 @@ env:
 permissions:
   id-token: write
   contents: read
+  issues: write
```

Comment:
Is write a superset of read & write? Presumably we need to do both, right?

Reply:
Write is what they actually have in their readme to use